developer-workflow/development-cycle.rst

```diff
@@ -77,11 +77,17 @@ releases; the terms are used interchangeably. These releases have a
 **micro version** number greater than zero.
 
 The only changes allowed to occur in a maintenance branch without debate are
-bug fixes.  Also, a general rule for maintenance branches is that compatibility
+bug fixes, test improvements, and edits to the documentation.
+Also, a general rule for maintenance branches is that compatibility 
 must not be broken at any point between sibling micro releases (3.5.1, 3.5.2,
 etc.).  For both rules, only rare exceptions are accepted and **must** be
 discussed first.
 
+Back-porting changes reduces the risk of future conflicts.
```

Comment:
Little nit: we use the closed, hyphenless version elsewhere, let's do the same here:

```suggestion
Backporting changes reduces the risk of future conflicts.
```